src/App.js

```diff
@@ -1,29 +1,6 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
+import './components/calculator.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default function App() {
+  return <Calculator />;
```

Comment:
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/270064624-2dd6af12-81ad-4e18-b7ec-a6d1007c5475.png)|![image](https://user-images.githubusercontent.com/99715640/270064691-553e408f-7029-4bda-bccb-876689204b13.png)|

- [ ] Good job with the component creation but it looks like your UI is not exactly as the project requirement at the moment. It would be a lot more visually appealing and aesthetically impressive if the calculator is not taking the entire screen. Kindly make the width something like the design provided - maybe 30 - 50% of the entire screen size.
- [ ] Also, it appears your display screen is missing the default output which signals your calculator is `on`.